tests/mtest/src/main/scala/tests/BaseSuite.scala

```diff
@@ -37,6 +37,21 @@ class BaseSuite extends TestSuite {
   def beforeAll(): Unit = ()
   def afterAll(): Unit = ()
   def intercept[T: ClassTag](exprs: Unit): T = macro Asserts.interceptProxy[T]
+  def isValidScalaVersionForEnv(scalaVersion: String): Boolean =
+    ScalaVersion
+      .fromString(scalaVersion)
+      .fold(
+        (e: ScalaVersion.ScalaVersionParseError) => this.fail(e.errorMessage),
+        (sv: ScalaVersion) => ScalaVersion.validScalaVersionForCurrentJDK(sv)
+      )
+
+  def guardOnScalaVersion(scalaVersion: String)(f: => Unit): Unit =
```

Comment:
nit: let's remove this helper and use `if (isValidScalaVersion(..)` instead

Reply:
Once we decide on what to do here (https://github.com/scalameta/metals/pull/1171#discussion_r357985278) I'll remove this.

Reply:
Updated.